internal/system/agent/execution_engine.go

```diff
@@ -0,0 +1,60 @@
+package agent
+
+import (
+	"fmt"
+	"github.com/edgexfoundry/edgex-go/internal/system/agent/logger"
+	"os/exec"
+)
+
+type ExecuteApp struct {
+}
+
+func runExec(service string, operation string) error {
+
+	// Preparing the call to the executor app.
+	cmd := exec.Command(Configuration.ExecutorPath, service, operation)
+
+	cmd.Dir = Configuration.ExecutorPath
+
+	_, err := cmd.CombinedOutput()
+	if err != nil {
+		logs.LoggingClient.Error(fmt.Sprintf("the error occurred in the invocation of executor: %v ", err.Error()))
+	} else {
+		logs.LoggingClient.Info("invocation of executor succeeded")
```

Comment:
It would be nice if this message could contain what service the executor handled and what operation it did with it. But this can be filed as an issue and handled in a followup PR.